.bazelrc

```diff
@@ -18,6 +18,8 @@ build --incompatible_strict_action_env
 build --host_force_python=PY3
 build --host_javabase=@bazel_tools//tools/jdk:remote_jdk11
 build --javabase=@bazel_tools//tools/jdk:remote_jdk11
+build --java_runtime_version=remotejdk_11
```

Comment:
I'm not particularly familiar with the Java ecosystem but know that 17 is the latest version. Should we upgrade to `remote_jdk17` as per https://docs.bazel.build/versions/main/user-manual.html#flag--java_runtime_version. If so probably need to adjust https://github.com/envoyproxy/envoy/blob/main/bazel/BUILD#L619-L620 as well (and maybe the Kafka extension)

Reply:
could we change java separate to bazel - im just thinking less complexity, and easier to revert etc

Reply:
Fair enough